### PR TITLE
[17.0][MIG] auto_backup: Migration to 17.0

### PR DESCRIPTION
**Pull request by rpinset**

### CODE REVIEW

**Comment by rpinset** [comment]
@rvalyi I followed the wiki though https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#how-to

these are my only modifications
https://github.com/rpinset/server-tools/commit/b5eb35a138a874f142029d6d3879c393644d501a

File: auto_backup/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2004-2009 Tiny SPRL (<http://tiny.be>).
+# Copyright 2015 Agile Business Group <http://www.agilebg.com>
+# Copyright 2016 Grupo ESOC Ingenieria de Servicios, S.L.U. - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Database Auto-Backup",
+    "summary": "Backups database",
+    "version": "16.0.1.0.0",
```
Comment (xtanuiha) [inline auto_backup/__manifest__.py]: version should be "17.0.1.0.0"

Comment (rpinset) [reply auto_backup/__manifest__.py]: My bad
I forget look this
Thanks 
It's fixed

**Review by CRogos — COMMENTED** [review]
Could you solve the pre-commit issues? 
You can also squash translation commits of the same person & language:
image

**Comment by rpinset** [comment]
> Could you solve the pre-commit issues? You can also squash translation commits of the same person & language: image

Sorry @CRogos, but this is my first attempt to push a migration.
I don't know how to use pre-commit.
I followed the wiki though https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#how-to
these are my only modifications
https://github.com/rpinset/server-tools/commit/b5eb35a138a874f142029d6d3879c393644d501a
and https://github.com/OCA/server-tools/pull/2894/commits/cafb66fb0cbdee05a1969c790ffa459c9cbfd190

**Comment by CRogos** [comment]
Yes I also struggled a lot with my first migration. 
pre-commit is a tool to automate certain things like code formatting. in this case, one issue is the xml formatting. 
But instead of changing it manually you should install pre-commit on your dev environment. 
https://github.com/OCA/maintainer-tools/wiki/Install-pre-commit
image

To squash certain commits, you find the instructions here: 
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

in short the most important commands are:
git config --global --add rebase.instructionFormat "(%an <%ae>) %s"
git rebase -i origin/17.0
\# change "pick" by "f" where needed and commit the changes
git push -f

**Comment by rpinset** [comment]
@CRogos I started again and ended up getting all the checks green 😊
can you check this ?

**Comment by CRogos** [comment]
yes indeed. 

But could you still squash some commits. e.g. when there are translations of the same person and same language.
image

All migrations commits should be merged and instead of merging 17.0 into this branch you should have rebased this branch. 
image

Maybe you can rebase it again to remove the merge commit?

**Comment by rpinset** [comment]
My bad, I forgot. I'm going to try that

**Comment by CRogos** [comment]
No need to close it... just rebase and update the branch with push force.

**Comment by rpinset** [comment]
> No need to close it... just rebase and update the branch with push force.

this was an automatic consequence of my git cli :(
i try again https://github.com/OCA/server-tools/pull/2975

**Comment by rpinset** [comment]
this was an automatic consequence of my git cli :(
i try again here https://github.com/OCA/server-tools/pull/2975
